src/encoding.ts

```diff
@@ -75,10 +75,11 @@ export interface Encoding<F> {
   size?: FieldDefWithCondition<MarkPropFieldDef<F>> | ValueDefWithCondition<MarkPropFieldDef<F>>;
 
   /**
-   * The symbol's shape (only for `point` marks). The supported values are
+   * For `point` marks the supported values are
    * `"circle"` (default), `"square"`, `"cross"`, `"diamond"`, `"triangle-up"`,
    * or `"triangle-down"`, or else a custom SVG path string.
    * __Default value:__ If undefined, the default shape depends on [mark config](config.html#point-config)'s `shape` property.
+   * For `geoshape` marks it should be a fielddef of the geojson data
```

Comment:
No description should be added below "__Default value:__" line

Could you please move it above and add a line break between the main description and the  "__Default value:__"  (Yeah we should have added it prior to this PR.)


fielddef => field definition